databricks_cli/stack/api.py

```diff
@@ -77,10 +82,10 @@ def deploy(self, config_path):  # overwrite to be added
         stack_config = self._load_json(config_path)
         status_path = self._generate_stack_status_path(config_path)
         stack_status = self._load_json(status_path)
-        new_stack_status = self.deploy_config(stack_config, stack_status)
+        new_stack_status = self.deploy_config(stack_config, stack_status, overwrite)
```

Comment:
here we may have more options in the future which may make the interface complex. How about just pass a dictionary such as `options = {overwrite: True}`?

Reply:
Yeah I'll look into using **kwargs